plugin/ale.vim

```diff
@@ -120,10 +122,10 @@ let g:ale_warn_about_trailing_whitespace =
 
 " Housekeeping
 
-augroup ALECleanup
+augroup ALECleanupGroup
     autocmd!
     " Clean up buffers automatically when they are unloaded.
-    autocmd BufUnload * call ale#cleanup#Buffer('<abuf>')
+    autocmd BufUnload * call ale#cleanup#Buffer(expand('<abuf>'))
```

Comment:
:+1: I missed that when I first added it. We definitely need to add that test for cleanup.